memcached-tool-ng

```diff
@@ -270,7 +270,7 @@ elsif ( $mode eq "display" ) {
 	foreach my $p ( keys %{$defaultStats} ) {
 		$it->{$p} = $defaultStats->{$p} if ! defined $it->{$p};
 	}
-        $totMemory += $it->{total_pages}*$item_size_max;
```

Comment:
Hey Jai! Thanks for your contribution. Just out of curiosity, did you see the comment on line #263 about the `-I` parameter? Is that outdated? Thanks.